crates/test-utils/src/minicore.rs

```diff
@@ -181,7 +181,7 @@ pub mod convert {
     }
     // endregion:as_ref
     // region:infallible
-    pub enum Infallibe {}
+    pub enum Infallible {}
```

Comment:
One thing I just realized regarding minicore, we should have a test that verifies minicore contains no unresolved things somehow to guard against accidental typos. (Again something for a follow up)

Reply:
Good idea. Long term I think we also need something to make sure it is in sync with real core, or reusing the real core somehow.